Templates/BaseGame/game/tools/assetBrowser/scripts/assetTypes/shape.tscript

```diff
@@ -318,16 +318,23 @@ function AssetBrowser::buildShapeAssetPreview(%this, %assetDef, %previewData)
                            "Shape File path: " @ %assetDef.getShapeFile();
                            
    if(%this.selectMode)
+   {
       %previewData.doubleClickCommand = "AssetBrowser.selectAsset( AssetBrowser.selectedAsset );";
+   }
    else
+   {
+      if(EditorSettings.value("Assets/Browser/doubleClickAction", "Edit Asset") $= "Edit Asset")
       %previewData.doubleClickCommand = "AssetBrowser.editAsset( "@%assetDef@" );";
+      else
+         %previewData.doubleClickCommand = "AssetBrowser.onShapeAssetEditorDropped( "@%assetDef@" );";
```

Comment:
Generally I think if you don't have braces it should be a single line. I'd prefer braces here